sunpy/net/hek2vso/__init__.py

```diff
@@ -1,10 +1,8 @@
 """
-This module provides a translation layer between the HEK and the VSO.
-It allows you to acquire records of data that are available via the
-VSO, based on the data in HEK event entries.
-
-.. warning::
-    This module is in beta and maybe unstable.
+This module provides a translation layer between the HEK and the VSO. It allows
+you to acquire records of data that are available via the VSO, based on the
+data in HEK event entries.
 
+.. warning::     This module is in beta and maybe unstable.
```

Comment:
!!!  Doesn't reformatting the `warning` directive like this cause it to fail?

Reply:
Yeah,  it can't handle rst directeives.